mkinclude/govc.mk

```diff
@@ -5,8 +5,8 @@ uname_m = $(shell uname -m)
 
 GOVC_URL ?= $(VSPHERE_USER):$(VSPHERE_PASSWORD)@$(VSPHERE_SERVER)
 
-# Path to Terraform binary.
-GOVC ?= tmp/govc.bin
+# Path to govc binary.
+export GOVC ?= tmp/govc.bin
```

Comment:
IMO this is not needed as the whole import.sh can be moved into a makefile target